.storybook/themes/commerce.scss

```diff
@@ -627,6 +627,23 @@
 						),
 					),
 				),
+				product-information: (
+					components: (
+						heading: (
+							color: var(--global-black),
+							font-size: var(--global-font-size-9),
+							font-weight: var(--global-font-weight-7),
+						),
+						stack: (
+							gap: var(--global-spacing-2),
+						),
+						price: (
+							color: var(--text-color),
+							font-size: var(--global-font-size-7),
+							font-weight: var(--global-font-weight-4),
+						),
+					),
+				),
```

Comment:
do you have the mobile version of these fonts? looks like 20px in the producti Info block. I could be looking at the wrong designs 
![Screen Shot 2022-07-21 at 06 26 51](https://user-images.githubusercontent.com/5950956/180192462-3aacb15d-3f3f-4e63-bc3e-f26426bdf08c.png)